src/Shell/I18nShell.php

```diff
@@ -87,7 +87,7 @@ public function init($language = null)
             $language = $this->in('Please specify language code, e.g. `en`, `eng`, `en_US` etc.');
         }
         if (strlen($language) < 2) {
-            return $this->abort('Invalid language code. Valid is `en`, `eng`, `en_US` etc.');
+            $this->abort('Invalid language code. Valid is `en`, `eng`, `en_US` etc.');
```

Comment:
Yes, a newline would do the trick.
Also, isn't it now just an exception being thrown? That should be easier to work with now in general, I guess.